renovate.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "schedule:earlyMondays"
```

Comment:
Can you align with https://github.com/canonical/lxd/blob/main/.github/renovate.json